JhipsterXamarin/UnitTestJhispterXamarin/TestLogin.cs

```diff
@@ -0,0 +1,64 @@
+﻿using JhipsterXamarin.Models;
+using JhipsterXamarin.Services;
+using JhipsterXamarin.ViewModels;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using MvvmCross.Navigation;
+using MvvmCross.Tests;
+
+namespace UnitTestJhispterXamarin
+{
+    [TestClass]
+    public class TestLogin : MvxIoCSupportingTest
+    {
+        LoginViewModel loginViewModel;
+        LoginModel loginModel = new LoginModel();
+
+        [TestMethod]
+        public void TestViewModel()
+        {
+            initializeAll();
+        }
+        public void initializeAll()
+        {
+            base.Setup(); // from MvxIoCSupportingTest
+            AdditionalSetup();
+            loginViewModel.Initialize();
+        }
+        protected override void AdditionalSetup()
+        {
+            var mockAuthService = new Mock<IAuthenticationService>();
+            var mockNavLoginService = new Mock<IMvxNavigationService>();
+            loginViewModel = new LoginViewModel(mockNavLoginService.Object, mockAuthService.Object);
+
+            Ioc.RegisterSingleton<IAuthenticationService>(mockAuthService.Object);
+            Ioc.RegisterSingleton<IMvxNavigationService>(mockNavLoginService.Object);
+        }
+        [TestMethod]
+        public void TestSignInAdmin()
+        {
+            initializeAll();
```

Comment:
you must use [TestInitiliaze] or [ClassInitialize] => https://www.automatetheplanet.com/mstest-cheat-sheet/